### DISCUSSION TITLE
SOLT -  Twelve Term Difference

### DISCUSSION THREAD

**Discussion by jobonny** [scikit-rf/scikit-rf#997] [Q&A] [answered]
Hello everyone,

I'm using scikit-rf  v0.30.0 to calibrate some simulations I got from Ansys HFSS for a GSG probe. I've been applying TwelveTerm correction, but I tried SOLT recently and got some different results. Now I'm scratching my head, wondering if I'm overlooking something. Is SOLT treating the Thru as a Flush? Any way to avoid this? And could this be the reason behind the difference in magnitude for S11 and S22?

Thanks for your help!
Giorgio

![Magnitude_SOLT_TwelveTerm](https://github.com/scikit-rf/scikit-rf/assets/144179941/c147a561-119f-4d30-8be1-39f8fa0a99d1)
![Phase_SOLT_TwelveTerm](https://github.com/scikit-rf/scikit-rf/assets/144179941/d8d42320-ce06-4c44-b02e-3b025b87cf29)
[SOLT_TwelveTerm.zip](https://github.com/scikit-rf/scikit-rf/files/13867610/SOLT_TwelveTerm.zip)

**Comment by Vinc0110** [comment]
Hi,
there has been an update on this in PR #990. I'm not an expert on this, but have a look yourself. Maybe this will be resolved with the next package release, which will include PR #990.

**Comment by arsenovic** [comment, marked as answer]
if you have  known  'ideals' for all standards, then use twelveterm. 

SOLT was originally intended to be  an implicit flush thru standard set.
SOLT inherits twelveterm now. 

let us know  if you have any more questions
